docs/export.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+"""Script to export the ReDoc documentation page into a standalone HTML file."""
+import json
+import sys
+import os
+from pathlib import Path
+
+HTML_TEMPLATE = """<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <title>My Project - ReDoc</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="shortcut icon" href="https://fastapi.tiangolo.com/img/favicon.png">
+    <style>
+        body {
+            margin: 0;
+            padding: 0;
+        }
+    </style>
+    <style data-styled="" data-styled-version="4.4.1"></style>
+</head>
+<body>
+    <div id="redoc-container"></div>
+    <script src="https://cdn.jsdelivr.net/npm/redoc/bundles/redoc.standalone.js"> </script>
+    <script>
+        var spec = %s;
+        Redoc.init(spec, {}, document.getElementById("redoc-container"));
+    </script>
+</body>
+</html>
+"""
+
+if __name__ == "__main__":
+    os.environ["ERT_STORAGE_DATABASE_URL"] = "sqlite:///ert.db"
```

Comment:
How about we use the in-memory database? `"sqlite://"`should suffice [1]

[1] https://docs.sqlalchemy.org/en/14/dialects/sqlite.html#connect-strings